quilt ps

sirspinach opened the pull request:
`quilt ps` looks like this:

```
 % quilt ps      
MACHINE        ROLE        PROVIDER      REGION         SIZE          CONNECTED
Machine-2      Master      Amazon        us-west-1      m4.large      true
Machine-3      Worker      Amazon        us-west-1      m4.large      true

CONTAINER      MACHINE      IMAGE         COMMAND    LABELS
Container-1    Machine-3    nginx:1.10    ""         web_tier
```

mmussomele commented on quiltctl/command/ps.go:
```diff
@@ -0,0 +1,65 @@
+package command
```
Now that we have `quilt ps`, why not ditch `quilt containers` and `quilt machines`?

ejj replied:
yep definitely should ditch them

sirspinach commented:
* added Usage text for `ps`, `machines`, `containers`, and `daemon`.
   * Added Stitch ID row to the front of `quilt containers` because we use Stitch ID for `quilt exec`. 

`quilt ps` now looks like this:
```
MACHINE      ROLE      PROVIDER    REGION       SIZE        CONNECTED
Machine-5    Master    Amazon      us-west-1    m4.large    true
Machine-6    Worker    Amazon      us-west-1    m4.large    true

STITCH ID    CONTAINER      MACHINE      IMAGE         COMMAND    LABELS
1            Container-5    Machine-6    nginx:1.10    ""         web_tier
```

ejj commented:
I'm happy with this, will merge when travis gives the OK
